Required/Scans.ipf

```diff
@@ -1086,11 +1089,18 @@ function ScanMultipleK2400LS625Magnet2D(keithleyIDs, start, fin, numptsx, delayx
 end
 
 
-
-function ScanLS625Magnet(instrID, startx, finx, numptsx, delayx, [y_label, comments, nosave]) 
-	variable instrID, startx, finx, numptsx, delayx,  nosave
+function ScanLS625Magnet(instrID, startx, finx, numptsx, delayx, [y_label, comments, nosave, fast]) //set fast=1 to run quickly
+	variable instrID, startx, finx, numptsx, delayx,  nosave, fast
 	string y_label, comments
-	//abort "WARNING: This scan has not been tested with an instrument connected. Remove this abort and test the behavior of the scan before running on a device!"	
+	
+	
+	variable ramprate
+	
+	if(paramisdefault(fast))
+		fast=0
+	endif
+	
+	
```

Comment:
In IGOR if the parameter is default it is already set to 0. But it is good to be explicit. 
A nice 1 liner would be:
fast = paramisdefault(fast) ? 1 : fast